packages/dds/tree/.vscode/Tree.code-workspace

```diff
@@ -21,13 +21,15 @@
 		"typescript.preferences.autoImportSpecifierExcludeRegexes": [
 			// Avoid suggesting imports from parent index file, which almost always is a mistake which creates a cyclic dependency.
 			// Importers in almost all cases should import it from a sibling instead (the same place the index file imports it from).
-			"^\\.\\/index\\.js",
+			"^\\.\\/index\\.js$",
 			// Same justification as the ./index.js case, but catches imports from parents further up the directory hierarchy.
-			"\\.\\.\\/index\\.js",
+			// Rejects cases like ../index.js and ../../index.js.
+			"\\.\\.\\/index\\.js$",
 			// Avoid suggesting imports which reach into a directory, unless the import is for the index file.
 			// Often tests for a specific file will want to violate this, so violating this could occasionally be useful,
 			// but removing it from an auto-import suggestion in all other cases is helpful.
-			"\\/.*\\/(?!index\\.js$)"
+			// Rejects cases like ./core/treeNodeSchema.js while allowing ./core/index.js, ../util.js and ./util.js.
+			"\\/[^.]*\\/(?!index\\.js$)"
```

Comment:
This one actually had a bug. It used to match ../util.js and ./util.js which we do allow. It has been updated to allow reaching into paths that contain a literal `.` to fix that.